packages/rum-core/test/performance-monitoring/capture-navigation.spec.js

```diff
@@ -23,21 +23,26 @@
  *
  */
 
+import * as navTiming from '../../src/performance-monitoring/navigation/navigation-timing-spans'
```

Comment:
this has been extremely helpful to test the **baseTime** logic 🎉 , which it was not tested before this PR